pkg/tee/key.go

```diff
@@ -0,0 +1,48 @@
+package tee
+
+import (
+	"encoding/base64"
+	"os"
+
+	"github.com/edgelesssys/ego/ecrypto"
+)
+
+var (
+	KeyDistributorPubKey string
+	SealingKey           string
+)
+
+func LoadKey(datadir string) error {
+	key, err := os.ReadFile(datadir + "/sealing_key")
+	if err != nil {
```

Comment:
Ditto as above. Minor: in all of this file, can we enrich the errors a bit to provide more context as to when/where they happened?